vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/ContextMenuIT.java

```diff
@@ -34,8 +34,9 @@ public void documentScroll_overlayPosition() {
         // Make sure to use a small enough viewport so that the document needs
         // to be scrolled to see the cell
         getDriver().manage().window().setSize(WINDOW_SIZE_SMALL);
-        // Scroll the document horizontally
-        executeScript("document.documentElement.scrollLeft = 100");
+        // Scroll the document so the cell is visible
```

Comment:
The viewport here has now less height as well, added some scrolling to make the cell visible.